src/pages-and-resources/discussions/data/api.js

```diff
@@ -0,0 +1,101 @@
+import { getConfig } from '@edx/frontend-platform';
```

Comment:
Much of this file is new code related to the POST request.  Prior to this, that was all mocked.

Reply:
You'll notice lots of TODOs.  This is the remainder of the work to get this whole thing working.